e2e/libs/workload/workload.py

```diff
@@ -23,6 +23,8 @@ def get_name_suffix(*args):
 def create_storageclass(name):
     if name == 'longhorn-test-strict-local':
         filepath = "./templates/workload/strict_local_storageclass.yaml"
+    elif name == 'longhorn-test-nfs-options':
+        filepath = "./templates/workload/storageclass-nfs-options.yaml"
```

Comment:
Tested based on https://github.com/longhorn/longhorn-tests/pull/1805.

`storageclass-nfs-options.yaml` should be `storageclass_nfs_options.yaml`.
Others LGTM.

Reply:
I'll make that change, thanks!.  Hopefully I can run this myself from Jenkins?

Reply:
Yes, we can run customized e2e tests by building images by yourself now, but since e2e test has been refactored by https://github.com/longhorn/longhorn-tests/pull/1604, it is possible that you need to make extra changes to make things work.